code/web/src/modules/user/Profile.js

```diff
@@ -41,13 +42,15 @@ const Profile = (props) => (
 
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
-        <p style={{ color: grey2, marginBottom: '0.5em' }}>{props.user.details.email}</p>
-        <p style={{ color: grey2, marginBottom: '3em' }}>{props.user.details.address}</p>
-        <img src={routeImage + props.user.details.image} alt={props.user.details.name} style={{ width: 400 }}/>
-        <p style={{ color: black, marginBottom: '2em', marginTop: '2em' }}>{props.user.details.bio}</p>
-
+        {props.user.details ? 
```

Comment:
What situations was this conditional added for?

Reply:
Sometimes the user data would get dropped from the store, this redirects the user to login if that ever happens

Reply:
That is a good fallback.

Reply:
I think that is a good fallback to have